personalizing_conda_on_gyre_sverdrup.md

```diff
@@ -81,8 +42,7 @@ dependencies:
 This is all you need to do. Now you can open notebooks that have access to this environment, as discussed next.
 
 **Opening a notebook with your environment**
-After you have created your own environment, you should be able to access it as kernel in your jupyter lab. It will show up on your launcher: 
-![](https://i.imgur.com/v64ppy2.png)
+After you have created your own environment, you should be able to access it as kernel in your jupyter lab. You can do this by opening a notebook and then selecting your new Kernel from the dropdown in the top right. Your environment may take a minute or so to propagate to this dropdown after you install it. 
 
```

Comment:
Should we add the picture that was there before back in? Or update to a new one?

Reply:
It will not appear in the launcher, so the old image is inappropriate. I'm not sure we need a new one, but can do if you want.

Reply:
Thanks for clarifying. I always like pictures in guides, but its up to you. Ill merge either way.